operator/pkg/ciliumendpointbatch/aggregator.go

```diff
@@ -0,0 +1,281 @@
+// Copyright 2021 Authors of Cilium
```

Comment:
Plan is to replace this with a workqueue, correct?

Reply:
I'll explore using `workqueues` for reconciling the CEB updates to api-server. I

Reply:
I think it should, the entire aggregator logic is going to be replaced by workqueue, I don't think we need to reinvent the wheels in this PR. @varunmar